src/geotools/pom.xml

```diff
@@ -49,8 +49,17 @@
           <groupId>javax.media</groupId>
           <artifactId>jai_core</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.postgresql</groupId>
+          <artifactId>postgresql</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>42.1.1</version>
```

Comment:
is this going to have an impact on geoserver at all?  Will this change drivers in the plugin?  
What happens if we deploy postgis and geogig?
(this might not be an issue)